### ISSUE TITLE
simulation will not start

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1594]
```
Please complete the following lines...

Application Version: 5.3.0 parallel
SVN Revision Number:3193
Compile Date:fri, 30 jan 2009
Operating System:win 7 64

Describe details of the issue below:
the simulation will not start and does not give me an error code or anything it just
says error message and then exited without calling finalize
```

Original issue reported on code.google.com by `rbruce@fpc-consultants.com` on 2012-04-23 22:12:23

<hr>
- _Attachment: [765500_01_A.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1628/comment-0/765500_01_A.fds)_

**Comment by gforney** [comment]
```
The case runs for me (FDS 5.5.3, 64 bit linux). Have you ever run FDS in parallel before?
If not, check the User's Guide and documentation for PyroSim. Let me know if you get
it to work.
```

Original issue reported on code.google.com by `mcgratta` on 2012-04-24 12:25:21

**Comment by gforney** [comment]
```
Issue 1629 has been merged into this issue.
```

Original issue reported on code.google.com by `mcgratta` on 2012-04-24 13:56:04

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2012-05-15 20:54:31
